src/main/java/com/findork/chiriezerie/controller/ApartmentController.java

```diff
@@ -0,0 +1,53 @@
+package com.findork.chiriezerie.controller;
+
+import com.findork.chiriezerie.exception.AppException;
+import com.findork.chiriezerie.model.Apartment;
+import com.findork.chiriezerie.model.daos.ApartmentDao;
+import com.findork.chiriezerie.service.IApartmentService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+@RestController
+@RequestMapping("/api/apartments")
+@AllArgsConstructor
+public class ApartmentController {
+
+    private final IApartmentService apartmentService;
+
+    @GetMapping("/")
```

Comment:
Remove the ("/") because the client is calling "api/apartments" and it receives a 404. Same for the POST request.